src/main/resources/liquibase/workbench_master.xml

```diff
@@ -58,5 +58,6 @@
 	<include file="workbench_changelog/18_2_0.xml" relativeToChangelogFile="true"/>
 	<include file="workbench_changelog/18_3_0.xml" relativeToChangelogFile="true"/>
 	<include file="workbench_changelog/18_4_0.xml" relativeToChangelogFile="true"/>
+	<include file="workbench_changelog/19_1_0.xml" relativeToChangelogFile="true"/>
```

Comment:
18_4_1

Reply:
Yes please move to `18_4_1` file and modify the changeset ID to `v18.4.1-1`